framework/src/loops/ComputeJacobianThread.C

```diff
@@ -152,14 +149,29 @@ ComputeJacobianThread::subdomainChanged()
   // If users pass a empty vector or a full size of vector,
   // we take all kernels
   if (!_tags.size() || _tags.size() == _fe_problem.numMatrixTags())
+  {
     _warehouse = &_kernels;
+    _dg_warehouse = &_dg_kernels;
```

Comment:
Explain to me why we have these two different members? I'm sure there's a reason; I'd just like to understand why

Reply:
Aw, the `*_warehouse` is dynamic and is a subset of the `*_kernels`. 👍

Reply:
One is static and the other changes always.